app/brave_main_delegate.cc

```diff
@@ -147,6 +148,9 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       autofill::features::kAutofillServerCommunication.name,
       features::kAudioServiceOutOfProcess.name,
       features::kDefaultEnableOopRasterization.name,
+      // TODO(darkdh): remove this when USS bookmarks is mature
```

Comment:
is this still needed after https://github.com/brave/brave-browser/pull/4551 and https://github.com/brave/brave-core/pull/2471 ?

Reply:
it is not necessary after  brave/brave-browser#4551 and #2471
removed in https://github.com/brave/brave-core/pull/2010/commits/2cb26c9cd3bc2059439fc3700c94ca56132e438d